### ISSUE TITLE
Does Native Control (no DCC) Still Work in macOS 15.0 Sequoia?

### ISSUE THREAD

**Issue by l33** [MonitorControl/MonitorControl#1709]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

So far I have been using MonitorControl 4.2.0 with macOS 14 Sonoma and was quite happy using the native control to control the brightness of multiple LG Ultrafine displays at once. With the update to macOS 15.0 Sequoia I also had to update MonitorControl because it kept crashing. With its latest 4.3.3 release native brightness control no longer seems to work. I have to use DCC and I am not too happy about it:

1. DCC seems to use a range of brightness, for example it allows me to completely turn off the display which I do not want.
2. Every time my Mac wakes from sleep the brightness is set to whatever the native control was set to before changing the brightness using DCC. I effectively have to change the brightness again after every wake.

Is the brightness control using the native interface broken in macOS 15.0 Sequoia or am I overlooking something? Was the fix for 4.2.0 effectively to disable native brightnes control?

**Comment by waydabber** [comment]
No, everything works fine with Sequoia, except if your display is in HDR mode. When it is, the app will treat Sequoia's SDR peak brightness remap control as native brightness control and use that instead of DDC. If you want to continue using HDR mode and DDC at the same time, you need to switch to BetterDisplay. Make sure you enable DDC in HDR mode under the display's settings.
